Modules/_ctypes/callproc.c

```diff
@@ -1123,6 +1123,15 @@ PyObject *_ctypes_callproc(PPROC pProc,
                 goto cleanup; /* leaking ? */
             }
         }
+#if (defined(__x86_64__) && (defined(__MINGW64__) || defined(__CYGWIN__))) || \
+        defined(__aarch64__)
+        void *tmp;
```

Comment:
I don't really agree--I find variable declarations in an if statement pretty ugly, but I can't disagree it at least limits it scope (even if it's ultimately irrelevant).  Maybe I'll change it if/when there are other more substantive changes to make.